core/src/main/java/org/apache/accumulo/core/client/SampleNotPresentException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.accumulo.core.client;
+
+public class SampleNotPresentException extends RuntimeException {
+
+  public SampleNotPresentException(Exception e) {
```

Comment:
Looking at your rendered Gist of the README, this leaks out Thrift information back to the user. I think it would be better to extract meaningful information to the user about what failed than our RPC representation. "org.apache.accumulo.core.client.SampleNotPresentException: TSampleNotPresentException(extent:TKeyExtent(table:32, endRow:null, prevEndRow:null))". For example, we can try to turn the table ID back into a table name and maybe include the actual extent (since it's public api now) for advanced debugging purposes?